doc/ruby/options.md

```diff
@@ -0,0 +1,723 @@
+# Ruby Command-Line Options
+
+## About the Examples
+
+Some examples here use command-line option `-e`,
+which passes the Ruby code to be executed on the command line itself:
+
+```sh
+$ ruby -e 'puts "Hello, World."'
+```
+
+Some examples here assume that file `desiderata.txt` exists:
+
+```
+$ cat desiderata.txt
+Go placidly amid the noise and the haste,
+and remember what peace there may be in silence.
+As far as possible, without surrender,
+be on good terms with all persons.
+```
+
+## Options
+
+### `-0`: \Set `$/` (Input Record Separator)
+
+Option `-0` defines the input record separator `$/`
+for the invoked Ruby program.
+
+The optional argument to the option must be octal digits,
+each in the range `0..7`;
+these digits are prefixed with digit `0` to form an octal value.
+
+If no argument is given, the input record separator is `0x00`.
+
+If an argument is given, it must immediately follow the option
+(no intervening whitespace or equal-sign character `'='`);
+argument values:
+
+- `0`: the input record separator is `''`;
+  see {Special Line Separator Values}[rdoc-ref:IO@Special+Line+Separator+Values].
+- In range `(1..0377)`:
+  the input record separator `$/` is set to the character value of the argument.
+- Any other value: the input record separator is `nil`.
+
+Examples:
+
+```sh
+$ ruby -0 -e 'p $/'
+"\x00"
+ruby -00 -e 'p $/'
+""
+$ ruby -012 -e 'p $/'
+"\n"
+$ ruby -015 -e 'p $/'
+"\r"
+$ ruby -0377 -e 'p $/'
+"\xFF"
+$ ruby -0400 -e 'p $/'
+nil
+```
+
+See also:
+
+- {Option -a}[options_md.html#label-a-3A+Split+Input+Lines+into+Fields]:
```

Comment:
```suggestion
- {Option -a}[rdoc-ref:@a-3A+Split+Input+Lines+into+Fields]:
```

Reply:
This link did not work for me.  Try as I did, I could not find any rdoc-ref that would work with a heading that begins with a hyphen.

On my machiine:

```
$ rdoc --version
6.3.1
$ ruby --version
ruby 3.0.2p107 (2021-07-07 revision 0db68f0233) [x86_64-linux-gnu]
```

Reply:
Use the latest and bundled version.

Reply:
@nobu, what version of RDoc do you have where that link works?

Here's what I have (on WIndows), where it does not work.

```
$ ruby --version
ruby 3.3.0 (2023-12-25 revision 5124f9ac75) [x64-mingw-ucrt]
$ rdoc --version
6.5.0
```

Reply:
Also does not work with rdoc version 6.6.2 on Windows.  @peterzhu2118, does the link from @nobu work for you (and, regardless, what version are you running)?

Reply:
No, but this works for me:

```suggestion
- {Option -a}[rdoc-ref:ruby/options.md@a-3A+Split+Input+Lines+into+Fields]:
```

Reply:
Fixed (all such).